helpers/checkpoint_logger/__init__.py

```diff
@@ -17,79 +17,22 @@
 )
 
 import sentry_sdk
-from shared.metrics import Counter, Histogram, metrics
+from shared.metrics import metrics
```

Comment:
I guess you can also use the opportunity to remove all these statsd metrics

Reply:
I have a followup PR incoming, I'll remove them in that one 👌